janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java

```diff
@@ -67,30 +67,30 @@ public void testPrintSchemaElements() {
         GraphOfTheGodsFactory.load(graph);
         mgmt = graph.openManagement();
 
-        String expected = "99EB9D82CE85D15A9120E31969B3F861";
+        String expected = "A18422278042949EE2B162837EC27A63";
```

Comment:
Do you know why these hashes changed?

Reply:
Yes. It also prints some index order. Which was `incr` and `decr` and now it is `asc` and `desc`. That is why the output hash has been changed